torchaudio/models/wavernn.py

```diff
@@ -347,6 +355,68 @@ def forward(self, waveform: Tensor, specgram: Tensor) -> Tensor:
         # bring back channel dimension
         return x.unsqueeze(1)
 
+    @torch.jit.export
+    def infer(self, specgram: Tensor) -> Tensor:
+        r"""Inference method of WaveRNN.
+
+        This function currently only supports multinomial sampling, which assumes the
+        network is trained on crossentropy loss.
```

Comment:
```suggestion
        network is trained on cross_entropy loss.
```

Reply:
Thanks for the comment, I've fixed it [here](https://github.com/pytorch/audio/pull/1704/files#diff-7f4ad259da182d3840fe4c75dd0c0b7b9fe30f31503df0eff87b1e5c9378cac4R363)